frontend/src/components/Playback/Playback.js

```diff
@@ -105,7 +104,11 @@ const Playback = ({
                         pauseAudio(playConfig);
                         return;
                     }
-                    let latency = playAudio(playConfig, sections[index]);
+                    // if the current player has resume_play set to true,
+                    // retrieve previous player's decisionTime from sessionStorage
+                    const playheadShift = playConfig.resume_play ? 
+                        parseFloat(window.sessionStorage.getItem('decisionTime')) : 0;
```

Comment:
I think it would be better to make a dedicated helper class or function for fetching and storing values from and to the session storage. It could be in this component, or perhaps in the `util` folder if we think we're going to use it more often than once.

Added bonus is that we can spy on that helper method to see if it gets called in our unit tests.